core/embed/rust/src/ui/display/font.rs

```diff
@@ -22,10 +29,10 @@ pub struct Glyph {
     pub bearing_x: i16,
     /// Top-side vertical bearing
     pub bearing_y: i16,
-    data: &'static [u8],
+    data: &'a [u8],
 }
 
-impl Glyph {
+impl<'a> Glyph<'a> {
```

Comment:
mucking with pointers reduced but  `fn GlyphData::get_glyph_data` needs to `offset` into `FontInfo::glyph_data` which is `*const *const u8` from C. So I moved the calculation of total size into a separate function `GlyphData::calculate_glyph_size` and the calculation is now not done in `Glyph::load` because it takes a slice which has `len()`.

The result is two `unsafe` blocks in `get_glyph_data`

See: cadcede0b73ebecaac79d3ea3e01acbbdb77912b